sanskrit_parser/base/sanskrit_base.py

```diff
@@ -76,10 +76,12 @@ def __init__(self, thing=None, encoding=None, unicode_encoding='utf-8', strict_i
             if thing is not None:
                 # Autodetect Encoding
                 self.encoding = SCHEMES[detect.detect(self.thing)]
-        if not strict_io:
-            # Convert to SLP1 and normalize
-            self.thing = self.canonical()
+        if self.encoding != SLP1:
+            # Convert to SLP1 
```

Comment:
There is a trailing space on this line which is the cause of the lint error.

Reply:
Ah, stupid me. I should've run flake8 locally.